axelor-base/src/main/resources/views/Partner.xml

```diff
@@ -480,6 +480,10 @@
           onClick="save,action-partner-view-create-opportunity" link="" colSpan="3"/>
         <panel-include view="incl-opportunities-partner-form"/>
       </panel>
+      <panel name="contractsPanel" title="Contracts" showIf="id != null"
+        if="__config__.app.isApp('contract')">
+        <panel-include view="incl-contracts-partner-form"/>
+      </panel>
```

Comment:
Maybe it's not done like this for the other, but when it's possible, please use extension. You have no access to contract module in base so it's better to extend partner-form in Contract module.